app/models/spree/return_authorization.rb

```diff
@@ -64,6 +64,11 @@ def returnable_inventory
       order.shipped_shipments.collect{ |s| s.inventory_units.to_a }.flatten
     end
 
+    # Used when Adjustment#update! wants to update the related adjustment
+    def compute_amount(*_args)
+      amount.abs * -1
```

Comment:
and why not simply `-amount.abs`?

Reply:
I pulled it in directly from a Spree commit. I'll update it :+1:

Reply:
I figured. Don't worry, I'm not judging you :joy: